en/backup-to-gcs-using-br.md

```diff
@@ -0,0 +1,259 @@
+    - `spec.br.clusterNamespace`: The `namespace` of the cluster to be backed up.
+    - `spec.br.logLevel`: The log level (`info` by default).
+    - `spec.br.statusAddr`: The HTTP port used for listening a process status in BR to facilitate debugging. If not specified, BR does not listen the process status by default.
+    - `spec.br.concurrency`: The number of threads used by each TiKV process during backup. Defaults to `4` for backup and `128` for restore.
+    - `spec.br.rateLimit`: The speed limit, in MB/s. If set to `4`, the speed limit is 4 MB/s. The default is no limit.
+    - `spec.br.checksum`: Whether to verify the files after the backup is completed. Defaults to `true`.
+    - `spec.br.sendCredToTikv`: Whether the BR process passes its GCP privileges to the TiKV process. Defaults to `true`.
+    </details>
+
+    This example backs up all data in the TiDB cluster to GCS. Some parameters in `spec.gcs` can be ignored, such as `location`, `objectAcl`, and `storageClass`.
+
+    The `projectId` in the configuration is the unique identifier for the user project on GCP. For how to obtain the identifier, see [GCP Documentation](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+    <details>
+    <summary>Configure <code>storageClass</code>.</summary>
+
+    GCS supports the following `storageClass` types:
+
+    * `MULTI_REGIONAL`
+    * `REGIONAL`
+    * `NEARLINE`
+    * `COLDLINE`
+    * `DURABLE_REDUCED_AVAILABILITY`
+
+    If you do not configure `storageClass`, the default is `COLDLINE`. See [GCS Documentation](https://cloud.google.com/storage/docs/storage-classes) for details.
+    
+    </details>
+
+    <details>
+    <summary>Configure the object access-control list (ACL) strategy</summary>
+
+    CS supports the following ACL strategies:
+
+    * `authenticatedRead`
+    * `bucketOwnerFullControl`
+    * `bucketOwnerRead`
+    * `private`
+    * `projectPrivate`
+    * `publicRead`
+
+    If you do not configure `storageClass`, the default is `COLDLINE`. See [GCS Documentation](https://cloud.google.com/storage/docs/storage-classes) for details.
+    </details>
+
+2. After creating the `Backup` CR, use the following command to check the backup status:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl get bk -n test1 -owide
+    ```
+
+<details>
+
+<summary>More descriptions of fields in the <code>Backup</code> CR</summary>
+
+* `.spec.metadata.namespace`: The namespace where the `Backup` CR is located.
+* `.spec.from.host`: The address of the TiDB cluster to be backed up.
+* `.spec.from.port`: The port of the TiDB cluster to be backed up.
+* `.spec.from.user`: The accessing user of the TiDB cluster to be backed up.
+* `.spec.gcs.bucket`: The name of the bucket which stores data.
+* `.spec.gcs.prefix`: This field is used to make up the path of the remote storage: `s3://${.spec.gcs.bucket}/${.spec.gcs.prefix}/backupName`. This field can be ignored.
```

Comment:
Should be `gcs://`, @shuijing198799 please confirm the format.

Reply:
yes here should be `gcs://`